lib/action_policy/rspec/pundit_syntax.rb

```diff
@@ -43,6 +43,6 @@ def self.included(base)
   config.include(
     ActionPolicy::RSpec::PunditSyntax::PolicyExampleGroup,
     type: :policy,
-    example_group: {file_path: %r{spec/policies}}
+    parent_example_group: {file_path: %r{spec/policies}}
```

Comment:
TBH, I don't remember why I chose this pattern.
Shouldn't `file_path:  %r{spec/policies}` without any nesting work? E.g.:

```ruby
config.include(
  ActionPolicy::RSpec::PunditSyntax::PolicyExampleGroup,
  type: :policy,
  file_path: %r{spec/policies}
)
```

Reply:
@palkan not sure, I was just following what an automated warning shared with me. 🤷‍♂

Reply:
Let's change to a shorter version (above) then. I believe it should work.